app/bundles/LeadBundle/Entity/DoNotContact.php

```diff
@@ -67,7 +67,7 @@ public static function loadMetadata(ORM\ClassMetadata $metadata): void
 
         $builder->setTable('lead_donotcontact')
             ->setCustomRepositoryClass(\Mautic\LeadBundle\Entity\DoNotContactRepository::class)
-            ->addIndex(['reason'], 'dnc_reason_search');
+            ->addIndex(['lead_id', 'channel', 'reason'], 'dnc_reason_search');
```

Comment:
```suggestion
            ->addIndex(['lead_id', 'channel', 'reason'], 'leadid_reason_channel');
```
The name should be same https://github.com/mautic/mautic/pull/13413/files#diff-6a40dd21e0b34d38b0c7b9b0e5c046ca0e7de1057ec431b84f6b03dfb1a43e29R55

Reply:
My bad, I didn't notice!
I'll fix this